README.md

```diff
@@ -10,11 +10,17 @@
 
 - **Applicable**: View Panel and Listing View
 
+- [Release Notes](./widget/release_notes.md)
```

Comment:
Can we change the compatibility to 7.4.3 and later. As we have only tested from 7.4.3 this time.

Reply:
Done